src/com/facebook/buck/python/make_pex.py

```diff
@@ -27,20 +28,8 @@
         'third-party/py/twitter-commons/src/python'))
     sys.path.insert(0, os.path.join(
         buck_root, 'third-party/py/setuptools'))
-    pkg_resources_py = os.path.join(
-        buck_root,
-        'third-party/py/setuptools/pkg_resources.py')
-
-# Otherwise, we're running from a PEX, so import the `pkg_resources`
-# module via a resource.
-else:
-    import pkg_resources
-    pkg_resources_py_tmp = tempfile.NamedTemporaryFile(
-        prefix='pkg_resources.py')
-    pkg_resources_py_tmp.write(
-        pkg_resources.resource_string(__name__, 'pkg_resources.py'))
-    pkg_resources_py_tmp.flush()
-    pkg_resources_py = pkg_resources_py_tmp.name
+
+import pkg_resources
```

Comment:
One thing I'm not sure about (although I guess there's no change here) is how good the standalone pex.py isolates itself from the enclosing environment - i.e. if the local system has a pkg_resources, do we actually do enough to make sure that the correct one is loaded? I think some site-packages magic might cause pkg_resources to be imported with sys so we'd always pick up the system one.